src/kbmod/trajectory_generator.py

```diff
@@ -53,7 +49,12 @@ def create_trajectory_generator(config):
     logger = logging.getLogger(__name__)
     logger.info(f"Creating trajectory generator of type {name}")
 
-    return TrajectoryGenerator.generators[name](**config)
+    # Add any keyword arguments to the params, overriding current values.
+    params = copy.deepcopy(config)
+    params.update(kwargs)
+    logger.debug(str(params))
+
+    return TrajectoryGenerator.generators[name](**params, work_unit=work_unit)
```

Comment:
I had no expectation of this being a part of this PR, I was just thinking aloud of what is the most generic common interface. Nice!